.github/workflows/test.yml

```diff
@@ -30,5 +43,5 @@ jobs:
 
       - name: Run Forge tests
         run: |
-          forge test -vvv
+          forge test --fork-url ${{secrets.RPC_OP}} -vvv --via-ir
```

Comment:
didn't know about the test's `fork` flag, does this mean the tests are being run on the testnet?

Reply:
It means that, for the test scope, we will fork the network's state. We are not running it in the testnet, but locally.